components/gitpod-db/src/typeorm/entity/db-workspace-instance-usage.ts

```diff
@@ -17,6 +17,21 @@ export class DBWorkspaceInstanceUsage {
     @Index("ind_attributionId")
     attributionId: string;
 
+    @Column("varchar")
+    userId: string;
+
+    @Column("varchar")
+    workspaceId: string;
+
+    @Column("varchar")
+    projectId: string;
+
+    @Column("varchar")
+    workspaceType: string;
```

Comment:
Is it important to match the column types with their occurences in other tables? I'm not sure they are always `varchar` in other tables.

Reply:
I had a look but couldn't find which one would not be a string. Do you know?

Reply:
* `workspaceId` is defined here like this: https://github.com/gitpod-io/gitpod/blob/60f4bad9db7a51f3479cfd1a53361f361f45305b/components/gitpod-db/src/typeorm/entity/db-workspace.ts#L27-L28
* `userId` like this: https://github.com/gitpod-io/gitpod/blob/60f4bad9db7a51f3479cfd1a53361f361f45305b/components/gitpod-db/src/typeorm/entity/db-user.ts#L16-L17
* `projectId` : https://github.com/gitpod-io/gitpod/blob/60f4bad9db7a51f3479cfd1a53361f361f45305b/components/gitpod-db/src/typeorm/entity/db-project.ts#L15-L16

Reply:
so they are all strings but with some extra annotations